File: 01-Backend/DatajudClient.API/DatajudClient.Domain/DTO/Tribunais/ReadTribunalDto.cs
using DatajudClient.Domain.DTO.Enderecos;

namespace DatajudClient.Domain.DTO.Tribunais
{
    public class ReadTribunalDTO
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Sigla { get; set; }
        public string Numero { get; set; }
        public string EndpointConsultaNumero { get; set; }
        public ReadCategoriaTribunalDTO Categoria { get; set; }
        public ReadEstadoDTO Estado { get; set; }


    }
}

File: 02-Frontend/DatajudClient.Front/src/app/dtos/tribunal/readTribunalDto.ts
import { ReadEstadoDto } from "../endereco/readEstadoDto";
import { ReadCategoriaTribunalDto } from "./readCategoriaTribunalDto";

export interface ReadTribunalDto {
  Id: number;
  Nome: string;
  Sigla: string;
  Numero: string;
  EndpointConsultaNumero: string;
  Categoria: ReadCategoriaTribunalDto;
  Estado: ReadEstadoDto;
}
